data/migrations/sql.jinja2

```diff
@@ -146,8 +146,12 @@ DECLARE
   row ALIAS FOR $1;
 BEGIN
   RETURN mz_calculate_min_zoom_pois_(
-{%- for param in layers['pois']['params'] %}
+{%- for param in layers['pois']['params'] -%}
+{%- if param.key == '"ele"' %}
+        row.tags->'ele',
+{%- else %}
```

Comment:
Ah, nice catch. I didn't realize that we had different schemas across tables. :+1: